tests/test_affinity.py

```diff
@@ -16,8 +16,8 @@ def test_affine_params(self):
         g = load_wkt('LINESTRING(2.4 4.1, 2.4 3, 3 3)')
         self.assertRaises(
             TypeError, affinity.affine_transform, g, None)
-        self.assertRaises(
-            TypeError, affinity.affine_transform, g, '123456')
+        # self.assertRaises(
+        #     TypeError, affinity.affine_transform, g, '123456')
```

Comment:
One more question: this no longer raises an error? (or a different error?)

Reply:
Fair question, it is not raised since numpy happily casts str to float (`np.array(list('123456'), dtype=float)`). This test was never really necessary, and on a dull day I would re-write all of the olden-days unittests to much more succinct pytest.

Reply:
Yes, let's just remove the commented code then